configs/indexes.json

```diff
@@ -189,5 +189,8 @@
 	],
 	"ZRX/USD": [
 		"json(https://api.binance.com/api/v1/klines?symbol=ZRXUSDT&interval=1d&limit=1).0.4"
+	],
+	"VIXEOD": [
+
```

Comment:
I'm not sure this is a good idea because now you're spreading out where the data is for all indexes, is there any reason the api endpoint can't be here? It seems like there is already custom handling for some indexes, this one seems to only require an API key substitution using environment variable as commented above

Reply:
I think ideally you'd want people to be putting their own in.   We don't want people to think that one specific API is what they're required to pull or use.

Reply:
Yep, a comment here can explain that the API can be replaced, but I also agree that a single file should be used to specify the APIs, with my comment from that last review I meant to put the API URL here with the following format
`"json(https://www.quandl.com/api/v3/datasets/CHRIS/CBOE_VX1.json?api_key=${VIXEOD_API_KEY}).dataset.data.0.4"` and `VIXEOD_API_KEY` is replaced from the .env file

Reply:
we don't want everyone using the same API's, so I don't think it's too much of an issue whether you have to manually be updating your config file or your indexes (I personally prefer config since it's shorter)

Reply:
> we don't want everyone using the same API's,

didn't understand this bit, can you explain. At the moment people use the indexes file and can change it their personal setup and preference. The config for me is to setup how the miner works and the indexes file is to setup how the trackers acquire data. Mixing both breaks the responsibilities separation. I think we should either use the config for everything or keep it separate as it is now - config for the miner setup and indexes for the trackers.


It would be confusing to spread tracker URLs in 2 places. this also add more complexity to the code to handle the paid vs unpaid APIs differently.

Reply:
Here is my idea for this. let me know if you see any problems with this solution.
https://github.com/tellor-io/TellorMiner/pull/254